bin/twemproxy_exporter

```diff
@@ -5,6 +5,10 @@ require 'prometheus/client/rack/exporter'
 require 'rack'
 require 'yaml'
 
+lib = File.expand_path('../../lib', __FILE__)
+$LOAD_PATH.unshift(lib) if File.directory?(lib) && !$LOAD_PATH.include?(lib)
```

Comment:
This code is similar to what you'd see in the gemspec when you run `bundle gem new_gem`, and it pretty common to find in gems. That said, I just tested this and it looks like it's unnecessary anyway.